api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/Parameter.scala

```diff
@@ -17,6 +17,6 @@
 
 package ai.h2o.sparkling.api.generation.common
 
-case class Parameter(swName: String, h2oName: String, defaultValue: AnyRef, dataType: DataType, comment: String)
+case class Parameter(swName: String, h2oName: String, defaultValue: AnyRef, dataType: Class[_], comment: String)
 
 case class DataType(name: String, fullName: String, isEnum: Boolean)
```

Comment:
Seems like `DataType` class is unused now

Reply:
Spot on, thanks!